jinja2/filters.py

```diff
@@ -567,6 +567,12 @@ def do_indent(
             rv += u'\n' + u'\n'.join(
                 indention + line if line else line for line in lines
             )
+            try:
+                # Unescape any strings that were escaped due to concatenation
+                rv = Markup(rv.unescape())
```

Comment:
Why doesn't this need to be done to the `if blank:` case above?

Reply:
You're right, this was just a case of me coding my fix to the use case with which I was familiar.